specification/resources/apps/models/apps_create_app_request.yml

```
@@ -1,6 +1,9 @@
 properties:
   spec:
     $ref: app_spec.yml
+  project_id:
+    type: string
+    description: The UUID of the project the app should be assigned.
```

```suggestion
    description: The ID of the project the app should be assigned to. If omitted, it will be assigned to your default project.
```

We should describe the behavior if it is omitted from the request. I assume it would be assigned to your default project?

---

That's a great change, thank you.